oslib.go

```diff
@@ -62,6 +62,7 @@ func osExecute(L *LState) int {
 	var procAttr os.ProcAttr
 	procAttr.Files = []*os.File{os.Stdin, os.Stdout, os.Stderr}
 	cmd, args := popenArgs(L.CheckString(1))
+	args = append([]string{cmd}, args...)
```

Comment:
I think that `args` has to include command name.

I read source code of exec.Command wrapping `os.StartProcess`

https://github.com/golang/go/blob/master/src/os/exec/exec.go#L125